### PR TITLE
reduce pyrsistent version

### PR DESCRIPTION
**Pull request by matthiasdiener**
0.19.x causes https://github.com/illinois-ceesd/mirgecom/issues/791

### CODE REVIEW

**Comment by kaushikcfd** [comment]
Probably a better fit in `inducer/loopy`? -- https://github.com/inducer/loopy/blob/d02872181494458a2a2cc67c035bba78742deb29/setup.py#L99

(Happy to cherry-pick until that's merged)

**Comment by matthiasdiener** [comment]
> Probably a better fit in `inducer/loopy`?

We could add it there too, but the only failures I observed are related to the `Tree` class which is only in this fork.

**Comment by matthiasdiener** [comment]
Turns out the just-released 0.19.2 also fixes this issue (cf. https://github.com/tobgu/pyrsistent/issues/263)

File: setup.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def write_git_revision(package_name):
           "codepy>=2017.1",
           "colorama",
           "Mako",
-          "pyrsistent",
+          "pyrsistent<0.19",
```
Comment (inducer) [inline setup.py:99]: FYI: There are two things that I think would improve this change.

- Add a link explaining why it's there.
- Write the constraint as `!=0.19.1` or some such, where the version is the one including the regression. That way, you don't preclude future updates when there's no reason to do so. (But, to be fair, you might have to update the constraint if further versions show up that include the regression.)

**Pull request merged**